clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -728,6 +728,16 @@ runSYCLPostLinkTool(ArrayRef<StringRef> InputFiles, const ArgList &Args) {
   SmallVector<StringRef, 8> CmdArgs;
   CmdArgs.push_back(*SYCLPostLinkPath);
   const llvm::Triple Triple(Args.getLastArgValue(OPT_triple_EQ));
+  Arg *SYCLDeviceLibLoc = Args.getLastArg(OPT_sycl_device_library_location_EQ);
+  if (SYCLDeviceLibLoc && !Triple.isSPIRAOT()) {
+    std::string SYCLDeviceLibLocParam = SYCLDeviceLibLoc->getValue();
+    std::string AssertDeviceLibLoc =
+        SYCLDeviceLibLocParam + "/libsycl-fallback-cassert.bc";
+    if (llvm::sys::fs::exists(AssertDeviceLibLoc)) {
```

Comment:
Do we need to 'assert' if the file is not present?

Reply:
Hi, @asudarsa 
I just fixed a little mistake here, we need to check the presence of "libsycl-native-bfloat16.bc" instead of assert devicelib file.

If the file is not present, we won't pass '--device-lib-dir=xxx' to sycl-post-link and sycl-post-link will skip embedding bfloat16 devicelib image, the whole compilation process will not crash or abort.

Thanks very much.